# Worksheet 1

This is for your joy, amusement, and edification. Don't think of it as an assignment. Think of it as a chance to exercise your skills. 

You may collaborate with friends, refer to notes, search the web, or use any other resources as you work on the problems below.

## Fun with math

1) Compute the sales tax on a item costing $12 assuming the tax rate is 7 percent. Leave the value in a variable named "tax"

In [4]:
# remember these are comments. Python will ignore them.
# write your code here
# it should start with
# tax =
import math
tax=0.07*12

2) Now, that you have the amount of the tax in a variable named tax. Compute the total purchase price into a variable named "total". 

In [6]:
# write your code here.
total=tax+12
print(total)

12.84


I think the total should be $12.84. Did you get that?

3) The number of permutations of length $n$ of a set with $k$ elements in $k^n$. Compute the number of permutations of length 10 of a set with 2 elements.

In [7]:
# write your code here. No need to assign the value to a variable but you may if you wish.
2**10


1024

I got 1024.

4) The number of atoms in the known universe is estimated to be approximately $4 \times 10^{81}$. Write that number in python's scientific notation. 

In [10]:
# write the number below this line.
numberofatoms= 4e81
print(numberofatoms)

4e+81


5) Compute the sum of the first 5 prime integers by simply writing out the expression. It will start like this: 2 + 3 + ...

In [10]:
# write the expression here
2+3+5+7+11

28

In [11]:
sum=2
sum= sum+3
sum= sum+5
sum= sum+7
sum= sum+11
print(sum)

28


6) Now compute that sum again but this time use a variable and only add one number at a time. For example your code might start like this:

    sum = 2
    sum = sum + 3
    ...

I got 28. Did you?

When you run the cell below you should get a **Submit this notebook** button. Click that button, login, and your notebook should be recorded.

In [2]:
# do not change the code below
import comp116

comp116.submit('ws1')